connection_params.ts

```diff
@@ -0,0 +1,59 @@
+import { env } from "deno";
+import { parseDsn } from "./utils.ts";
+
+const DEFAULT_CONNECTION_PARAMS = {
+    host: "127.0.0.1",
+    port: "5432",
+    user: "postgres",
+    database: "postgres",
+    password: "",
+    application_name: "deno_postgres"
+};
+
+export interface IConnectionParams {
+    database?: string;
+    host?: string;
+    port?: string;
+    user?: string;
+    password?: string;
+    application_name?: string;
+}
+
+export class ConnectionParams {
+    database?: string;
+    host?: string;
+    port?: string;
+    user?: string;
+    password?: string;
+    application_name?: string;
+    // TODO: support other params
+
+    constructor(config?: string | IConnectionParams) {
+        const envVars = env();
+
+        if (!config) {
+            config = {};
+        }
+        
+        if (typeof config === "string") {
+            const dsn = parseDsn(config);
+            if (dsn.driver !== "postgres") {
+                throw new Error(`Supplied DSN has invalid driver: ${dsn.driver}.`);
+            }
+
+            this.database = dsn.database || envVars.PGDATABASE;
```

Comment:
`getValue(dsn, "database", "PGDATABASE")` - should fallback to default value from `DEFAULT_CONNECTION_PARAMS`